src/pages/product/product.css

```diff
@@ -0,0 +1,217 @@
+:root {
```

Comment:
these lines (1:37) can be deleted since theese styles already get applied via the global.css file